### PR TITLE
Js add specs and cli docs

### PR DESCRIPTION
**Pull request by Jstein77**
## What are you changing in this pull request and why?
Adding additional generic example to metrics and measures docs.

### CODE REVIEW

File: website/docs/docs/build/metricflow-cli.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+description: "Query metrics and metadata in your dbt project with the metricflow cli"
+sidebar_label: "CLI"
+tags: [Metrics, Semantic Layer]
+---
+
+After you have defined metrics in your dbt project, you can easily query metrics, dimensions, dimension values, and validate your configs using the MetricFlow CLI.
+
+## Installation
+1. Create or activate your virtual environment.`python -m venv venv`
+2. Run `pip install dbt-metricflow`
```
Comment (mirnawong1) [inline website/docs/docs/build/metricflow-cli.md:16]: @Jstein77 is it `pip install dbt-metricflow` or `pip install metricflow` per the readme? 
 also what python versions are supported? the metricflow github repo says `requires-python = ">=3.8,<3.12"` but my error seems to imply it doesn't support 3.11, which I'm on?
 
I'm on python 3.11.4, so when i tried installing using `dbt-metricflow`, i got the below error:

```
(mwong-metrics) ➜  ~ git:(current) ✗ pip install dbt-metricflow
ERROR: Ignored the following versions that require a different python version: 0.0.1 Requires-Python <3.10,>=3.8; 0.0.2 Requires-Python <3.10,>=3.8; 0.0.3 Requires-Python <3.10,>=3.8
ERROR: Could not find a version that satisfies the requirement dbt-metricflow (from versions: none)
ERROR: No matching distribution found for dbt-metricflow
```

but when i used `pip install metricflow` as specified in the cli package github readme, it installed successfully. 


Screenshot 2023-07-19 at 10 08 54

File: website/docs/docs/build/metricflow-cli.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+## CLI commands
+
+## List
+Retrieve metadata values about metrics/dimensions/entities/dimension values
```
Comment (mirnawong1) [inline website/docs/docs/build/metricflow-cli.md]: adding an example here.  Does this retrieve measures?

```suggestion

This command retrieves metadata values related to [Metrics](/docs/build/metrics-overview), [Dimensions](/docs/build/dimensions), and [Entites](/docs/build/entities) values. 

For example, if you're trying to retrieve the `name` metadata value for a metric, you can use the following command:

```
mf list name
```

File: website/docs/docs/build/metrics-overview.md [hunk 3]
```diff
@@ -16,6 +16,20 @@ The keys for metrics definitions are:
 * `constraint`: For any type of metric, you may optionally include a constraint string, which applies a dimensional filter when computing the metric. You may think of this as your WHERE clause.  
 * `meta`: Additional metadata you want to add to your metric. 
 
+An example of the complete metrics spec configuration is below:
```
Comment (mirnawong1) [inline website/docs/docs/build/metrics-overview.md]: change to active voice

```suggestion
Here's a complete example of the metrics spec configuration:

```

File: website/docs/docs/build/metricflow-cli.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/build/metricflow-cli.md]: @Jstein77 committing this - hope that's ok!

File: website/docs/docs/build/metricflow-cli.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+## CLI commands
+
+## List
+
+This command retrieves metadata values related to [Metrics](/docs/build/metrics-overview), [Dimensions](/docs/build/dimensions), and [Entites](/docs/build/entities) values. 
+
+For example, if you're trying to retrieve the `name` metadata value for a metric, you can use the following command:
+
+
```
Comment (mirnawong1) [inline website/docs/docs/build/metricflow-cli.md:42]: is that right @Jstein77 ? 

```suggestion

```bash
mf list 

```
```

**Review by mirnawong1 — APPROVED** [review]
🚢

File: website/docs/docs/build/metricflow-cli.md (hunk 4 above)
Comment (Jstein77) [reply website/docs/docs/build/metricflow-cli.md:42]: No this wouldn't execute

Comment (mirnawong1) [reply website/docs/docs/build/metricflow-cli.md:42]: ah ok, what's would the correct command for `mf list` be @Jstein77 ?

Comment (Jstein77) [reply website/docs/docs/build/metricflow-cli.md:42]: mf list just shows you the available objects you can list by. You would need to run one of the commands below like `mf list metrics` to actually list the values. Running mf list just shows the help text

File: website/docs/docs/build/metricflow-cli.md (hunk 1 above)
Comment (Jstein77) [reply website/docs/docs/build/metricflow-cli.md:16]: I think this is a bug. The preferred way to install the cli is `pip install dbt-metricflow`

File: website/docs/docs/build/metricflow-cli.md (hunk 4 above)
Comment (Jstein77) [reply website/docs/docs/build/metricflow-cli.md:42]: I think we can just add `mf list`

Comment (mirnawong1) [reply website/docs/docs/build/metricflow-cli.md:42]: oh i gotcha, thanks!

File: website/docs/docs/build/metricflow-cli.md [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+- [`validate-configs`](#validate-configs) &mdash; Validates semantic model configurations.
+- [`health-checks`](#health-checks) &mdash; Performs data platform health check.
+- [`tutorial`](#tutorial) &mdash; Dedicated MetricFlow tutorial to help get you started.
+- [`query`](#query) &mdash; Query metrics and dimensions you want to see in the CLI. Refer to [query examples](#query-examples) to help you get started.
+
+## List
+
+This command retrieves metadata values related to [Metrics](/docs/build/metrics-overview), [Dimensions](/docs/build/dimensions), and [Entities](/docs/build/entities) values. 
+
+For example, if you're trying to retrieve the `name` metadata value for a metric, you can use the following command:
```
Comment (mirnawong1) [inline website/docs/docs/build/metricflow-cli.md]:
```suggestion
```

**Comment by mirnawong1** [comment]
let me know if you want me to merge @Jstein77 !

File: website/docs/docs/build/metricflow-cli.md (hunk 1 above)
Comment (Jstein77) [reply website/docs/docs/build/metricflow-cli.md:16]: https://github.com/dbt-labs/metricflow/pull/663 should be resolved when this is merged

**Pull request merged**